### PR TITLE
add travis-ci

### PR DESCRIPTION
**Pull request by isimonenko**

### CODE REVIEW

File: src/test/java/com/metamx/http/client/JankyServersTest.java [hunk 1]
```diff
@@ -246,18 +257,29 @@ public void testHttpsConnectionClosingServer() throws Throwable
...
       lifecycle.stop();
     }
+    System.out.println("Done testHttpsConnectionClosingServer");
+
+  }
+
+  public boolean isChannelClosedException(Throwable e)
+  {
+    return e instanceof ChannelException ||
+           (e instanceof IOException && e.getMessage().matches(".*Connection reset by peer.*"));
```
Comment (nishantmonu51) [inline src/test/java/com/metamx/http/client/JankyServersTest.java]: On travis, When an exception is thrown , in some cases it causes the channel to be closed with an IOException

File: src/test/java/com/metamx/http/client/JankyServersTest.java [hunk 2]
```diff
@@ -141,6 +142,8 @@ public static void tearDown() throws Exception
   @Test
   public void testHttpSilentServer() throws Throwable
   {
+    System.out.println("Starting testHttpSilentServer");
```
Comment (drcrallen) [inline src/test/java/com/metamx/http/client/JankyServersTest.java]: Why are these needed?

Comment (xvrl) [inline src/test/java/com/metamx/http/client/JankyServersTest.java]: if they're necessary, let's use log.debug

Comment (nishantmonu51) [inline src/test/java/com/metamx/http/client/JankyServersTest.java]: removed.
